src/services/api/createAppApiClient.ts

```diff
@@ -72,6 +74,32 @@ const updateUser =
       return res.data.data
     } catch (error) {}
   }
+
+type UploadImageResponse = {
+  status: boolean
+}
+const uploadImage =
+  (api: AxiosInstance) =>
+  async (req: FormData): Promise<UploadImageResponse | undefined> => {
+    try {
+      const res = await api.post<UploadImageResponse>("/user/me/avatar", req)
+
+      return res.data
+    } catch (error) {}
+  }
+type GetUserImageRequest = {
+  uid: string
```

Comment:
```suggestion
  userId: string
```
uid có nghĩa khác nhé